tpchgen/src/generators.rs

```diff
@@ -1723,13 +1724,14 @@ pub struct LineItem<'a> {
     /// Line item number within order
     pub l_linenumber: i32,
     /// Quantity ordered
+    // TODO: Spec has this as decimal.
     pub l_quantity: i64,
```

Comment:
`l_quantity` should be a decimal according to the spec, but switching it over broke some of the current tests against well-known output.

I didn't dig in too deeply, but I'll investigate at some point why that happens. But a quick poke at the trino source showed that they use long and don't format it like they do for the monetary fields, so figured that's probably baked into the output that's produced.

Reply:
FWIW the expected output I think also matches the original tpch dbgen program (as in I don't think this is a trino specific behavior). 

Read more about it here:
https://github.com/clflushopt/tpchgen-rs/tree/main/tpchgen/data#comparing-with-other-tpch-dbgen-programs

I also double checked it against the [original dbgen](https://github.com/electrum/tpch-dbgen) program I built from source this morning: